src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/study/StudyReportWeek.java

```diff
@@ -0,0 +1,90 @@
+package org.sagebionetworks.bridge.models.schedules2.adherence.study;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.joda.time.LocalDate;
+import org.sagebionetworks.bridge.models.schedules2.adherence.eventstream.EventStreamDay;
+import org.sagebionetworks.bridge.models.schedules2.adherence.weekly.WeeklyAdherenceReportRow;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonPropertyOrder({ "startDate", "endDate", "week", "adherencePercent", "rows", "byDayEntries", "type" })
+public class StudyReportWeek {
+    
+    private int weekInStudy;
```

Comment:
"week" looks like it should change to "weekInStudy".

Reply:
Good catch, changed